photosynthesis_metrics/utils.py

```diff
@@ -24,6 +24,7 @@ def _adjust_dimensions(input_tensors: Union[torch.Tensor, Tuple[torch.Tensor, to
 
 def _validate_input(
         input_tensors: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]],
+        supports_5d: bool,
```

Comment:
What do you think about default value? 5D size is used only in case of ssim, others do not support 5D.
```suggestion
        supports_5d: Optional[bool] = False,
```

Reply:
In general I avoid default values where possible because they enforce error-prone implicit behaviour